pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -24,6 +24,10 @@ let
     # to work with nix.
     # if true, the wrapper automatically appends those snippets when necessary
     , autoconfigure ? false
+
+    # append to PATH runtime deps of plugins
+    , autowrapRuntimeDeps ? false
```

Comment:
Why having it opt-in? We already apply a wrapper to each plugin, no?

Reply:
I thought we might have it opt-in at first to get feedback,to update the plugins with their runtimeDeps as well. And in a second step we would make it opt-out. We coul opt-in straightaway as well, I dont mind.

Reply:
one issue with the opt-out is that it forces us to update the doc in a way that works  with the legacy and new wrapper, which should be done but I am too lazy to do yet :'(  I would rather document the new wrapper in another PR

Reply:
I don't see a reason for changing plugins to use `runtimeDeps` in a subsequent PR. We can do it here to tackle problems before merging the feature. Unless you don't want to do this right now

But yeah, we can keep it opt-in for now, though I doubt someone would use opt-in undocumented features outside of [@]NixOS/neovim team